src/khoj/database/models/__init__.py

```diff
@@ -120,14 +120,23 @@ class OfflineChatProcessorConversationConfig(BaseModel):
     enabled = models.BooleanField(default=False)
 
 
+class SpeechToTextModelOptions(BaseModel):
+    class ModelType(models.TextChoices):
+        OPENAI = "openai"
+        OFFLINE = "offline"
+
+    model_name = models.CharField(max_length=200, default="whisper-1")
+    model_type = models.CharField(max_length=200, choices=ModelType.choices, default=ModelType.OPENAI)
+
+
 class ChatModelOptions(BaseModel):
     class ModelType(models.TextChoices):
         OPENAI = "openai"
         OFFLINE = "offline"
 
     max_prompt_size = models.IntegerField(default=None, null=True, blank=True)
     tokenizer = models.CharField(max_length=200, default=None, null=True, blank=True)
-    chat_model = models.CharField(max_length=200, default=None, null=True, blank=True)
+    chat_model = models.CharField(max_length=200, default="gpt-4")
```

Comment:
I don't think the default should be `gpt-4`. It's not certain the admin will set an open AI API key. The default could be the open source model, however, with a default of `ModelType.Offline` below. Otherwise, I'd just continue to leave this field mandatory but without a default.